packages/pg/lib/client.js

```diff
@@ -24,11 +24,9 @@ class Client extends EventEmitter {
 
     // "hiding" the password so it doesn't show up in stack traces
     // or if the client is console.logged
+    this.password = this.connectionParameters.password
```

Comment:
I don’t think these are worth changing, but that’s less important.

Reply:
this was just to reduce lines. I can make it the way it was, which seems very important when this library has gone through 8 major versions and leaks private keys.